src/groth16/aggregate/srs.rs

```diff
@@ -141,6 +146,10 @@ where
         let h_up = h_low + n;
         let g_alpha_powers_table =
             precompute_fixed_window(&self.g_alpha_powers[g_low..g_up], WINDOW_SIZE);
+        let g_alpha_powers_end_table = precompute_fixed_window(
```

Comment:
@cryptonemo @DrPeterVanNostrand this adds some overhead to srs in memory for the case this is not used, your call as to how much that is a realy issue/impact in the real world. 

This might be better to be lazyilzy generated to just avoid that problem

Reply:
If it's just one call to `precompute_fixed_window` that's extraneous, and you're not aggregating a large number of proofs, I think that it's probably ok

Reply:
Or did you mean that the `g_alpha_powers_end_table` vector itself is extraneous?

Reply:
Looks like the latter, as it was previously not used but is now proposed to be required.  I haven't gotten a measure of how much it affects things yet though (timing/RAM).